news/2018-05-14-twc-finals-recap.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "osu!taiko World Cup - Finals Week 1 Recap!"
+date: 2018-05-14 00:00:00 +0000
+---
+The drum mashing of the Taiko World Cup 2018 has concluded while the fruit puns of the osu!catch World Cup will soon commence. The journey for the gold medal has left us with one winning team, but definitely not in the dominant fashion that was expected. 
+
+![](/wiki/shared/news/2018-05-07-twc-finals-recap/podium.png)
+
+**Note: Due to an unfortunate situation surrounding foul play, team United States has forfeited out of the TWC and we won't go over their matches in this recap. You can read about the situation at https://osu.ppy.sh/community/forums/topics/743380**
+
+### Finals Matches
+
+**In the first match, Canada claimed their top 4 placing by taking down South Korea 7-3.** The Koreans dominated the speedy nomod picks [Nanoka - Tsukimade Todoke, Fushi no Kemuri](https://osu.ppy.sh/beatmapsets/763774#taiko/1605963) and [Lily - Scarlet Rose](https://osu.ppy.sh/beatmapsets/195238#taiko/463330), but that only got them so far. Having reached the Finals as the only remaining mid-seed, this hailed the end of South Korea's spectacular run this TWC. Just imagine what they could accomplish with a bigger roster in future cups...
+
+History was simultaneously repeated and made in the Winners Finals match between Japan vs. Taiwan in one of the most exciting matches we've seen thus far. Everyone was on the edge of their seats as the increasing chinks in Japan's armor turned into an unprecedented 3 - 3 tie halfway into the match. Japan however regained their composure and pulled ahead to end with a 7 - 3 victory. With Japan's armor not as impenetrable as we thought, we'll have to wait until the Grand Finals to see if any of the remaining teams can turn those visible chinks into a victory.
```

Comment:
```diff
- With Japan's armor not as impenetrable as we thought, we'll have to wait until the Grand Finals to see if any of the remaining teams can turn those visible chinks into a victory.
+ With Japan's armor not as impenetrable as we thought, the stage was set for the Grand Finals to be more exciting than we've seen in a long time.
```

Rephrased because Grand Finals already happened